addons/stock/tests/test_move2.py

```diff
@@ -2337,6 +2337,47 @@ def test_update_product_move_line_with_different_uom(self):
         self.assertTrue(float_is_zero(reserved_on_move_lines_LtDA - quant_LtDA.reserved_quantity, precision_digits=precision_digits))
         self.assertTrue(float_is_zero(reserved_on_move_lines_GtDA - quant_GtDA.reserved_quantity, precision_digits=precision_digits))
 
+class TestUnreserveOnlyRequiredQuantity(TestStockCommon):
```

Comment:
you don't need to create anew class for this, I think you can use the one already present in this file. 
Also as the feature is rather low level. I would only create 2 stock moves, no picking to keep the test small and quick. 
thank you

Reply:
Hello, i did the suggested changes! ;) 
what about the 16.0 branch ? this commit will be forwardport , right ?